docs/Overview.md

```diff
@@ -84,7 +84,7 @@ type Predicate interface {}
 // Storage provides an interface to the storage layer.
 type Storage interface {
     // Read gets data from the underlying storage system and returns a data frame or error state.
-    Read(context.Context, []Predicate, TimeRange, Grouping) (DataFrame, ErrorState)
+    Read(context.Context, []Predicate, TimeRange, Grouping) (Table, ErrorState)
```

Comment:
Referring to my previous comment, if we are intending to document the current state of things, then this would return an array of tables.